config/routes.rb

```diff
@@ -112,12 +112,9 @@ def not_implemented
 
     Feature.with(:pensions_and_retirement) do
       get LandingPagePaths.path(:retirements, :index, :en),     to: 'retirements#index'
+      get LandingPagePaths.path(:retirements, :index, :cy),     to: 'retirements#index'
       get LandingPagePaths.path(:retirements, :budgeting, :en), to: 'retirements#budgeting'
-
-      Feature.with(:welsh_retirement_landing_pages) do
-        get LandingPagePaths.path(:retirements, :index, :cy),     to: 'retirements#index'
```

Comment:
It's still there — just moved out of the feature block. See: https://github.com/moneyadviceservice/frontend/pull/1046/files#diff-21497849d8f00507c9c8dcaf6288b136R115